applications/spectrum_analyzer/spectrum_analyzer.c

```diff
@@ -0,0 +1,475 @@
+#include <furi.h>
+#include <furi_hal.h>
+#include <gui/gui.h>
+#include <input/input.h>
+#include <stdlib.h>
+#include "spectrum_analyzer.h"
+
+void spectrum_analyzer_draw_scale(Canvas* canvas, const SpectrumAnalyzerModel* model) {
+
+    #define FREQ_BOTTOM_Y 50
```

Comment:
Defines in the code section is a bad pattern, please move it to the beginning of the file.

Reply:
OK